### ISSUE TITLE
MSbuild failed with cpp error

### ISSUE THREAD

**Issue by xinkunZ** [microsoft/Terminal#463]
```
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2039: “function”: 不是“std”的成员 (编译源文件 ..\OutputCell.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
  D:\SOFT\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.20.27508\include\variant(31): note: 参见“std”的声明 (编译源文件 ..\OutputCell.cpp)
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2065: “function”: 未声明的标识符 (编译源文件 ..\OutputCell.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2275: “std::wstring_view”: 将此类型用作表达式非法 (编译源文件 ..\OutputCell.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
  D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): note: 参见“std::wstring_view”的声明 (编译源文件 ..\OutputCell.cpp)
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2065: “pfnFallback”: 未声明的标识符 (编译源文件 ..\OutputCell.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2182: “SetGlyphWidthFallback”: 非法使用“void”类型 (编译源文件 ..\OutputCell.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2039: “function”: 不是“std”的成员 (编译源文件 ..\OutputCellIterator.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
  D:\SOFT\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.20.27508\include\bitset(19): note: 参见“std”的声明 (编译源文件 ..\OutputCellIterator.cpp)
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2065: “function”: 未声明的标识符 (编译源文件 ..\OutputCellIterator.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2275: “std::wstring_view”: 将此类型用作表达式非法 (编译源文件 ..\OutputCellIterator.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
  D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): note: 参见“std::wstring_view”的声明 (编译源文件 ..\OutputCellIterator.cpp)
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2065: “pfnFallback”: 未声明的标识符 (编译源文件 ..\OutputCellIterator.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
D:\Workspace\Terminal\src\types\inc\GlyphWidth.hpp(14): error C2182: “SetGlyphWidthFallback”: 非法使用“void”类型 (编译源文件 ..\OutputCellIterator.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
D:\Workspace\Terminal\src\buffer\out\TextBuffer.hpp(146): error C2039: “function”: 不是“std”的成员 (编译源文件 ..\cursor.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
  D:\Workspace\Terminal\src\buffer\out\UnicodeStorage.hpp(22): note: 参见“std”的声明 (编译源文件 ..\cursor.cpp)
D:\Workspace\Terminal\src\buffer\out\TextBuffer.hpp(146): error C2061: 语法错误: 标识符“function” (编译源文件 ..\cursor.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
D:\Workspace\Terminal\src\buffer\out\textBuffer.hpp(146): error C2039: “function”: 不是“std”的成员 (编译源文件 ..\Row.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
  D:\Workspace\Terminal\src\buffer\out\UnicodeStorage.hpp(22): note: 参见“std”的声明 (编译源文件 ..\Row.cpp)
D:\Workspace\Terminal\src\buffer\out\textBuffer.hpp(146): error C2061: 语法错误: 标识符“function” (编译源文件 ..\Row.cpp) [D:\Workspace\Terminal\src\buffer\out\lib\bufferout.vcxproj]
```

**Comment by lizy14** [comment]
see #449.

**Comment by zadjii-msft** [comment]
Yep, this is fixed by #449
